libzkbob-rs/src/client/mod.rs

```diff
@@ -69,6 +71,16 @@ pub struct TransactionData<Fr: PrimeField> {
     pub out_hashes: SizedVec<Num<Fr>, { constants::OUT + 1 }>,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct TransactionInputs<Fr: PrimeField> {
+    #[serde(bound(serialize = "", deserialize = ""))]
```

Comment:
is it really needed? I see no problem with default serde behavior  here

Reply:
Most likely it's a copy-paste issue. Removed in https://github.com/zkBob/libzkbob-rs/pull/46/commits/c21ea801b890216200e3ed539fa6ea13e707f70c